### PR TITLE
Feat/15 use env variables for config to allow for cd closes #15

### LINKED ISSUE #15: Use env variables for config to allow for CD

### PR DESCRIPTION
**Pull request by tlmn**
load custom config from .env

### CODE REVIEW

File: build.cfg.ts [hunk 1]
```diff
@@ -1,7 +1,8 @@
-import { config as customConfig } from "./custom.cfg.js";
 import * as types from "./src/cfg/types";
 import { merge } from "lodash/fp";
 
+const customConfig = process.env.CUSTOM_CFG !== undefined ? JSON.parse(process?.env?.CUSTOM_CFG) : {};
```
Comment (nikonikoniko) [inline build.cfg.ts:4]: I like this better.

One thing to note - this config currently _will_ be built in to the build, so should not contain secrets.

File: package.json [hunk 2]
```diff
@@ -37,6 +37,7 @@
     "uuid": "^8.3.2"
   },
   "devDependencies": {
+    "@types/lodash": "^4.14.182",
```
Comment (nikonikoniko) [inline package.json:40]: theoretically should be separate PR :japanese_ogre:

File: build.cfg.ts (hunk 1 above)
Comment (tlmn) [reply build.cfg.ts:4]: @nikonikoniko Wasn't it being baked in in the previous solution too?

Comment (nikonikoniko) [reply build.cfg.ts:4]: yes it was, was a general note
